schema.json

```diff
@@ -136,6 +136,10 @@
           "type": "string",
           "description": "Icon resource identifier that can be used find the icon"
         },
+        "tint": {
+          "type": "string",
+          "description": "Specifies a tint for the icon. Supported formats:#RRGGBB, #AARRGGBB. Supported  values: red, blue, green, black, white, gray, cyan, magenta, yellow, lightgray, darkgray, grey, lightgrey, darkgrey, aqua, fuchsia, lime, maroon, navy, olive, purple, silver and teal"
```

Comment:
Insert Space after `:` here: `formats:#RRGGBB`
Replace double space with single space at `Supported  values`

(also applies to duplicate line below).

Reply:
Done